### ISSUE TITLE
[Telemetry] Report the policies that have add_field enabled.

### ISSUE THREAD

**Issue by nimarezainia** [elastic/elastic-agent#4213]

**Comment by ycombinator** [comment]
Hey @nimarezainia, would you mind fleshing out some details in the issue description?  Thanks!

**Comment by ycombinator** [comment]
@kpollich Do you think it makes sense for this issue to be transferred to the Kibana repo, since the implementation for the data tagging / add_field feature is being done entirely in Kibana now, e.g. https://github.com/elastic/kibana/pull/183563?

**Comment by kpollich** [comment]
> @kpollich Do you think it makes sense for this issue to be transferred to the Kibana repo, since the implementation for the data tagging / add_field feature is being done entirely in Kibana now, e.g. [elastic/kibana#183563](https://github.com/elastic/kibana/pull/183563)?

Yes this can move over to Kibana as we can report this data by querying Kibana objects.
